src/uct/ib/base/ib_iface.c

```diff
@@ -150,6 +150,12 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Which pkey value to use. Should be between 0 and 0x7fff.",
    ucs_offsetof(uct_ib_iface_config_t, pkey_value), UCS_CONFIG_TYPE_HEX},
 
+#if HAVE_IBV_EXP_RES_DOMAIN
+  {"DISABLE_RESOURCE_DOMAIN", "n",
```

Comment:
let's make it positive e.g "UCX_RESOURCE_DOMAIN=y"
to be same spirit as other parameters

Reply:
done